jvm/src/main/scala/org/scalacheck/Platform.scala

```diff
@@ -63,7 +63,7 @@ private[scalacheck] object Platform {
 
   def newInstance(name: String, loader: ClassLoader, paramTypes: Seq[Class[_]])(args: Seq[AnyRef]): AnyRef =
     if(!args.isEmpty) ???
-    else Class.forName(name, true, loader).newInstance.asInstanceOf[AnyRef]
+    else Class.forName(name, true, loader).getDeclaredConstructor(paramTypes:_*).newInstance().asInstanceOf[AnyRef]
```

Comment:
if `args` is empty, then isn't `paramTypes` empty too?

Reply:
Good point!  Lets try without it.